perf_test/sparse/KokkosSparse_par_ilut.cpp

```diff
@@ -441,7 +452,10 @@ int main(int argc, char** argv)
 
   // Set dependent defaults
   if (nnz_per_row == -1) {
-    nnz_per_row = rows / 100;
+    nnz_per_row = std::min(rows / 100, 50);
+  }
+  if (bandwidth == -1) {
+    bandwidth = std::max(2 * (int)std::sqrt(rows), nnz_per_row);
```

Comment:
```suggestion
    bandwidth = std::max(2 * (int)std::sqrt(rows), 2 * nnz_per_row);
```

If ``bandwidth == nnz_per_row``, the matrix generator will complain that
``0.7 * bandwidth < max_nnz_per_row``.

Reply:
@brian-kelley , good catch! Thanks!